PCbuild/fix_lineendings.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python3
+#
+# Fixes line endings of batch files to ensure they remain LF.
```

Comment:
Hmm.  I just checked it out and got the right line endings even on your new `find_msbuild.bat` file.  I think this file can just be removed and we hope `.gitattributes` makes Git do the right thing :)